test/unit/pipeline/ReorderBufferTest.cc

```diff
@@ -365,26 +365,26 @@ TEST_F(ReorderBufferTest, branch) {
 
```

Comment:
For the `branch` test, we should be doing `EXPECT_CALL` on the mock branch predictor throughout to ensure a) the correct functions are being called, and b) the expected values are being passed to said functions